ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/index.tsx

```diff
@@ -223,13 +235,44 @@ const RunComponent = () => {
                     // always run implicitly and not part of the jobs, so add that manually.
                     const jobNames = Object.keys(ortRun.jobs).concat('config');
 
-                    downloadLogs(ortRun.id, '', jobNames.join(','));
+                    downloadLogs(ortRun.id, level, jobNames.join(','));
                   }}
                   variant='outline'
-                  className='font-semibold text-blue-400'
+                  className={'rounded-r-none font-semibold text-blue-400'}
                 >
-                  Download log archive
+                  Download log archive, level={level}
```

Comment:
I propose to shorten this a bit (and make it read less technical) by saying `Download {level} log archive`.

Reply:
And just to confirm how this is working: Selecting a level from the drop-down does not start the download immediately, but updates the text of the button, and a user still needs to click the main button to start the download, correct?

Reply:
Correct, that's intentional.

Reply:
I'd make this work the way that clicking a level from the dropdown immediately starts the download. I think that if someone selects a level, they always want to download it immediately after, so the second click is not needed.

Reply:
> I'd make this work the way that clicking a level from the dropdown immediately starts the download.

I don't have a *very* strong opinion here, but *if* doing that IMO the "Choose the log level" title in the drop down should go away completely, and the individual entries need to be full titles instead of just the level name.

Reply:
> > I'd make this work the way that clicking a level from the dropdown immediately starts the download.
> 
> I don't have a _very_ strong opinion here, but _if_ doing that IMO the "Choose the log level" title in the drop down should go away completely, and the individual entries need to be full titles instead of just the level name.

Yeah, that would be ideal IMO.

- Download INFO log archive
  - Download ERROR log archive
  - Download WARN log archive
  - etc.

Reply:
It also should probably be somehow communicated that selecting a level downloads them for that level and all above. Not sure what the best way is.

Reply:
Maybe instead of e.g. "Download INFO *log* archive" say "Download INFO *level* archive" to emphasize it's a level that includes less verbose levels, too?